app/models/like.rb

```diff
@@ -0,0 +1,13 @@
+class Like < ActiveRecord::Base
+  belongs_to :user, class_name: 'User'
+  belongs_to :post, class_name: 'Post'
```

Comment:
- [ ] As stated by the previous reviewers, please make sure to have the association to a 'author' and the class_name: 'User` as it is, just like you did in the Post model. 🙏

Reply:
@MahmoudBakr23 [Here](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd_v1_1.png) it says Like should be user_id not author_id, so it should be user, not author

Reply:
Thank you for communicating with me 😄I can understand your point. However, isn't making more sense when an author creates a post and an author likes and comments as well? Please see the requirements [here](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project_v1_1.md#:~:text=Create%20at%20least%206%20posts%20comments%20for%20one%20of%20the%20posts%20you%20created%20by%20running%20the%20following%20code%3A) and see how a comment is created using author, and not user. 🙏

Reply:
@Javivy